core/web/user_controller.go

```diff
@@ -132,8 +140,12 @@ func (c *UserController) UpdateRole(ctx *gin.Context) {
 		return
 	}
 
-	user, err := c.App.SessionORM().UpdateRole(request.Email, request.NewRole)
+	user, err := c.App.AuthenticationProvider().UpdateRole(request.Email, request.NewRole)
 	if err != nil {
+		if errors.Is(err, clsession.ErrNotSupported) {
+			jsonAPIError(ctx, http.StatusBadRequest, errors.New("Action is unsupported with configured authentication provider"))
```

Comment:
Why don't we declare one variable for all the copies of this error?
```go
var errUnsupportedForAuth = errors.New("action is unsupported with configured authentication provider")
```

Reply:
Done 🙂